<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookEntry</class>
 <widget class="QWidget" name="BookEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_ID">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_ID"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_Name">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_Name"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_Birthday">
     <property name="text">
      <string>Birthday</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_Bithday"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_AddDate">
     <property name="text">
      <string>Add date</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_AddDate"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_Save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
